prince/Makefile

```diff
@@ -0,0 +1,66 @@
+# 
+# Copyright (C) 2009-2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=prince
+PKG_VERSION:=v0.4
+PKG_RELEASE:=2
```

Comment:
I think RELEASE 1 here is okay because you never distributed the other version, it was just in the code review